src/components/SignUp/SignUpForm.css

```diff
@@ -107,13 +107,8 @@
 	}
 }
 
-#name-input-helper-text { display: none; }
-#name-input:not(:focus):user-invalid ~ #name-input-helper-text { display: block; }
+#name-input ~ .helper-container { display: none; }
+#name-input:user-invalid:not(:focus) ~ .helper-container { display: flex; }
```

Comment:
Question:

Why are we switching not(:focus):user-invalid with user-invalid:not(:focus) ?